src/Mutate4l/Commands/PrePad.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Mutate4l.Cli;
+using Mutate4l.Core;
+using Mutate4l.Utility;
+
+namespace Mutate4l.Commands
+{
+    // Extracts a region of the current clips, effectively cropping them. If two params, start - duration is used, otherwise 0 - duration.
+    public class PrePadOptions
+    {
+        [OptionInfo(type: OptionType.Default, 1/2f)]
+        public decimal[] Lengths { get; set; } = { 1 };

```

Comment:
Seems like you're only using the first value here anyway, so just use a standard decimal here instead.

Reply:
okay. I /thought/ it maybe future-proofing but fair point. 👍